### PR TITLE
python3Packages.pyvisa: init at 1.11.3, python3Packages.pyvisa-py: init at 0.5.1

### PR DESCRIPTION
**Pull request by mvnetbiz**
###### Motivation for this change
Add python library for communicating with test instruments (oscilloscope, waveform generator, power supply, etc.)

###### Things done

Added pyvisa and pyvisa-py (pure python backend)
Tested connecting to instruments over IP and basic queries work.

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 123868` at 2fd5e8f0 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built successfully:</summary>
  <ul>
    <li>python38Packages.pyvisa</li>
    <li>python38Packages.pyvisa-py</li>
    <li>python39Packages.pyvisa</li>
    <li>python39Packages.pyvisa-py</li>
  </ul>
</details>

File: pkgs/development/python-modules/pyvisa-py/default.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  src = fetchFromGitHub {
+    owner = "pyvisa";
+    repo = "pyvisa-py";
+    rev = version;
+    hash = "sha256-V1BS+BvHVI8h/rynLnOHvQdIR6RwQrNa2p2S6GQug98=";
+  };
+
+  nativeBuildInputs = [
+    setuptools_scm
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyvisa-py/default.nix:24]:
```suggestion
    setuptools-scm
```

File: pkgs/development/python-modules/pyvisa-py/default.nix [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, setuptools_scm
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyvisa-py/default.nix:4]:
```suggestion
, setuptools-scm
```

File: pkgs/development/python-modules/pyvisa/default.nix [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, setuptools_scm
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyvisa/default.nix:4]:
```suggestion
, setuptools-scm
```

File: pkgs/development/python-modules/pyvisa/default.nix [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+  src = fetchFromGitHub {
+    owner = "pyvisa";
+    repo = "pyvisa";
+    rev = version;
+    hash = "sha256-Qe7W1zPI1aedLDnhkLTDPTa/lsNnCGik5Hu+jLn+meA=";
+  };
+
+  nativeBuildInputs = [
+    setuptools_scm
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pyvisa/default.nix:22]:
```suggestion
    setuptools-scm
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 123868 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>4 packages built:</summary>  <ul>    <li>python38Packages.pyvisa</li>    <li>python38Packages.pyvisa-py</li>    <li>python39Packages.pyvisa</li>    <li>python39Packages.pyvisa-py</li>  </ul></details>

**Pull request merged**